sdk/build.gradle

```diff
@@ -62,6 +62,7 @@ android {
 
 dependencies {
     implementation fileTree(dir: "libs", include: ["*.jar"])
+    implementation 'androidx.security:security-crypto-ktx:1.0.0'
```

Comment:
confirming that we in fact want `1.1.0-alpha05`, like [stripe](https://github.com/stripe/stripe-android/blob/281c10868e886b616cfeb55bc1f96d98ae5d71a4/dependencies.gradle#L26C1-L27C1):